WorkflowManager.Dockerfile

```diff
@@ -25,6 +25,10 @@ COPY . ./
 RUN echo "Building MONAI Workflow Manager..."
 RUN dotnet publish -c Release -o out --nologo src/WorkflowManager/WorkflowManager/Monai.Deploy.WorkflowManager.csproj
 
+RUN echo "Fetching mc executable for minio..."
```

Comment:
if workflow manage doesn't use mc, I vote not to include it and disable health check for admin

Reply:
here's an example

```
services.AddMonaiDeployStorageService(hostContext.Configuration.GetSection("InformaticsGateway:storage:serviceAssemblyName").Value, Monai.Deploy.Storage.HealthCheckOptions.ServiceHealthCheck);
```

Reply:
Ok cool, didn't realize you could disable it, changed now